package.json

```diff
@@ -77,6 +79,7 @@
     "@babel/preset-react": "^7.24.7",
     "@babel/register": "^7.24.6",
     "@happy-dom/jest-environment": "^16.6.0",
+    "@playwright/experimental-ct-react": "^1.50.1",
```

Comment:
This must match the Docker image version, I believe.

Reply:
Yes, it must. And it matches. If not, Playwright would fail with error telling it.